class-gf-akismet.php

```diff
@@ -0,0 +1,466 @@
+<?php
+
+namespace Gravity_Forms\Gravity_Forms_Akismet;
+
+defined( 'ABSPATH' ) || die();
+
+use GFForms;
+use GFAddOn;
+use GFCommon;
+use GFAPI;
+use GFFormsModel;
+use Akismet;
+use Gravity_Forms\Gravity_Forms_Akismet\Settings;
+
+// Include the Gravity Forms Add-On Framework.
+GFForms::include_addon_framework();
+
+/**
+ * Gravity Forms Akismet Add-On.
+ *
+ * @since     1.0
+ * @package   GravityForms
+ * @author    Gravity Forms
+ * @copyright Copyright (c) 2020-2021, Gravity Forms
+ */
+class GF_Akismet extends GFAddOn {
+
+	/**
+	 * Contains an instance of this class, if available.
+	 *
+	 * @since  1.0
+	 * @var    GF_Akismet $_instance If available, contains an instance of this class
+	 */
+	private static $_instance = null;
+
+	/**
+	 * Defines the version of the Gravity Forms Akismet Add-On.
+	 *
+	 * @since  1.0
+	 * @var    string $_version Contains the version.
+	 */
+	protected $_version = GF_AKISMET_VERSION;
+
+	/**
+	 * Defines the minimum Gravity Forms version required.
+	 *
+	 * @since  1.0
+	 * @var    string $_min_gravityforms_version The minimum version required.
+	 */
+	protected $_min_gravityforms_version = GF_AKISMET_MIN_GF_VERSION;
+
+	/**
+	 * Defines the plugin slug.
+	 *
+	 * @since  1.0
+	 * @var    string $_slug The slug used for this plugin.
+	 */
+	protected $_slug = 'gravityformsakismet';
+
+	/**
+	 * Defines the main plugin file.
+	 *
+	 * @since  1.0
+	 * @var    string $_path The path to the main plugin file, relative to the plugins folder.
+	 */
+	protected $_path = 'gravityformsakismet/akismet.php';
+
+	/**
+	 * Defines the full path to this class file.
+	 *
+	 * @since  1.0
+	 * @var    string $_full_path The full path.
+	 */
+	protected $_full_path = __FILE__;
+
+	/**
+	 * Defines the URL where this add-on can be found.
+	 *
+	 * @since  1.0
+	 * @var    string The URL of the Add-On.
+	 */
+	protected $_url = 'https://gravityforms.com';
+
+	/**
+	 * Defines the title of this add-on.
+	 *
+	 * @since  1.0
+	 * @var    string $_title The title of the add-on.
+	 */
+	protected $_title = 'Gravity Forms Akismet Add-On';
+
+	/**
+	 * Defines the short title of the add-on.
+	 *
+	 * @since  1.0
+	 * @var    string $_short_title The short title.
+	 */
+	protected $_short_title = 'Akismet';
+
+	/**
+	 * Defines if Add-On should use Gravity Forms servers for update data.
+	 *
+	 * @since  1.0
+	 * @access protected
+	 * @var    bool
+	 */
+	protected $_enable_rg_autoupgrade = true;
+
+	/**
+	 * Defines the capabilities needed for the Gravity Forms Akismet Add-On
+	 *
+	 * @since  1.0
+	 * @access protected
+	 * @var    array $_capabilities The capabilities needed for the Add-On
+	 */
+	protected $_capabilities = array(
+		'gravityforms_gravityformsakismet',
+		'gravityforms_gravityformsakismet_uninstall',
```

Comment:
Looks like I forgot to fix the caps issue when I first created the add-on. I have an [open PR to fix it in the generator](gravityforms/gravityformsaddongenerator/pull/3).

```suggestion
		'gravityforms_akismet',
		'gravityforms_akismet_uninstall',
```